hostdb/hostdb.go

```diff
@@ -58,7 +58,6 @@ func ForEachAnnouncement(b types.Block, height types.BlockHeight, fn func(consen
 type Interaction struct {
 	Timestamp time.Time
 	Type      string
-	Success   bool
```

Comment:
Not sure all of the extra unmarshaling (and complexity) is worth it? Not really that much of an argument but it makes it harder to manually query the database as well.

Reply:
I don't mind either way but considering that the flag wasn't implemented fully and @lukechampine mentioned that he wants the hostdb to be a dumb store I just moved it into the result.

To be fair the only valuable info from querying that db is probably the error anyway. Which we condensed into a `Success` field. So at the very least I'd change `Success` to `Error` to be able to query for specific failures.